lib/ansible/modules/web_infrastructure/ansible_tower/tower_credential.py

```diff
@@ -282,23 +282,24 @@ def main():
                 team = team_res.get(name=module.params.get('team'))
                 params['team'] = team['id']
 
-            params['inputs'] = {}
             if module.params.get('ssh_key_data'):
                 filename = module.params.get('ssh_key_data')
                 if not os.path.exists(filename):
                     module.fail_json(msg='file not found: %s' % filename)
                 if os.path.isdir(filename):
                     module.fail_json(msg='attempted to read contents of directory: %s' % filename)
                 with open(filename, 'rb') as f:
-                    params['inputs']['ssh_key_data'] = f.read()
+                    module.params['ssh_key_data'] = f.read()
```

Comment:
From a style standpoint, this should be done a different way (it mutates an input rather than making the change in a scoped variable) but if we need to get this in quickly (for the 2.5.0 and 2.4.4 releases) then I won't block on it.  Just something to remember to fix later.